git/essentials/features-i/reset-vs-revert.md

```diff
@@ -10,11 +10,12 @@ type: normal
 category: must-know
 
 aspects:
-  
   - deep
   - obscura
   - workout
 
+link:
+  - '[Difference between git reset --soft and --hard](https://stackoverflow.com/questions/3528245/whats-the-difference-between-git-reset-mixed-soft-and-hard){website}'
```

Comment:
I think "Difference between git reset --soft and --hard" is a bit too long. how does this sound?

```suggestion
  - '[git reset --soft vs --hard](https://stackoverflow.com/questions/3528245/whats-the-difference-between-git-reset-mixed-soft-and-hard){website}'
```